framework/framework.go

```diff
@@ -299,28 +321,6 @@ func ProcessUpdate(ctx context.Context, obj interface{}, resources []Resource) e
 	return nil
 }
```

Comment:
So Framework methods are grouped together and functions on the bottom of the file. Noting changed semantically.